default/stringtables/en.txt

```diff
@@ -16719,6 +16719,9 @@ GREAT_HAPPINESS_DESC
 ULTIMATE_HAPPINESS_DESC
 +++	Ultimate [[metertype METER_HAPPINESS]]: +[[value ULTIMATE_HAPPINESS_VAL]]
 
+INDEPENDENT_HAPPINESS_DESC
+⋄	Independent [[metertype METER_HAPPINESS]]: Stability unaffected by good, poor or disconnected capital supply connection
```

Comment:
Maybe `Baseline Stability unaffected by supply connections`

Reply:
Will change tommorow, going to sleep. I think eventually I want to link it to the actual penalties and just need a place holder

Reply:
I'm not certain whether you meant Independent Stability: Baseline Stability unaffected by supply connections
or Baseline Stability: unaffected by supply connections